noxfile.py

```diff
@@ -23,11 +24,24 @@
 
 REQUIRED_DOCKER_VERSION = "20.10.17"
 
-# Sets the default session to `--list`
-nox.options.sessions = []
+nox.options.sessions = ["open_docs"]
+
+# This is important for caching pip installs
 nox.options.reuse_existing_virtualenvs = True
 
 
+@nox.session()
+def open_docs(session: nox.Session) -> None:
+    """Open the webpage for the developer/contribution docs."""
+    dev_url = "http://localhost:8000/fides/development/developing_fides/"
+    prod_url = "https://ethyca.github.io/fides/stable/development/developing_fides/"
```

Comment:
is this the right URL? going to it gives me a note that the docs have moved. clicking on the link to where the docs have moved to doesn't bring me to the dev/contributor docs though

<img width="1334" alt="image" src="https://user-images.githubusercontent.com/24641006/233100994-3de20c33-b04b-42a0-9c28-a354740debb4.png">

Reply:
it doesn't exist yet because the PR hasn't been merged, but if you run the docs locally and use `nox -- dev` it'll take you to the local one :)